src/Microsoft.Fx.Portability/ObjectModel/NuGetPackageInfo.cs

```diff
@@ -48,4 +49,34 @@ public override int GetHashCode()
             return _hashCode;
         }
     }
+
+    public class NuGetPackageInfoComparer : IComparer<NuGetPackageInfo>
+    {
+        public int Compare(NuGetPackageInfo x, NuGetPackageInfo y)
+        {
+            if (x == null)
+            {
+                if (y == null)
+                {
+                    return 0;
+                }
+                return -1;
+            }
+            else
+            {
+                if (y == null)
+                {
+                    return 1;
+                }
+                if (x.AssemblyInfo.Equals(y.AssemblyInfo, StringComparison.OrdinalIgnoreCase))
```

Comment:
Is it possible for `x.AssemblyInfo` to be null? if so, there may be a NullReferenceException here.

Reply:
Fixed